src/components/Registration/ProducerRegistration.js

```diff
@@ -0,0 +1,1202 @@
+}) {
+  if (currentStep !== 3) {
+    return null;
+  }
+  return [
+    <div className={classes.subtitleText}>
+      Contact Information
+    </div>,
+    <Grid
+      container
+      spacing={2}
+      alignItems="center"
+    >
+      <Grid item xs={12}>
+        <FormControl className={classes.formControl} variant="outlined">
+          <InputLabel id="mutiple-name-label" required>Market</InputLabel>
+          <Select
+            id="demo-mutiple-name"
+            multiple
+            value={formState.market}
+            onChange={handleSelect}
+            input={<Input />}
+            MenuProps={MenuProps}
+            name="market"
+            label="Population Served"
+          >
+            {MARKETS.map((market) => (
+              <MenuItem
+                key={market}
+                value={market}
+              >
+                {market}
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid item xs={12}>
+        <FormControl className={classes.formControl}>
+          <InputLabel id="single-name-label" required>Farm Size</InputLabel>
+          <Select
+            labelId="simple-select-label"
+            id="demo-single-name"
+            value={formState.farmSize}
+            onChange={handleSelect}
+            name="farmSize"
+            label="Farm Size"
+            fullWidth
+          >
+            {FARMSIZE.map((size) => (
+              <MenuItem
+                key={size}
+                value={size}
+              >
+                {size}
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid item xs={12}>
+        <FormControl className={classes.formControl} variant="outlined">
+          <InputLabel id="mutiple-name-label" required>Farming Practices</InputLabel>
+          <Select
+            id="demo-mutiple-name"
+            multiple
+            value={formState.farmPractices}
+            onChange={handleSelect}
+            input={<Input />}
+            MenuProps={MenuProps}
+            name="farmPractices"
+            label="Population Served"
+          >
+            {PRACTICES.map((p) => (
+              <MenuItem
+                key={p}
+                value={p}
+              >
+                {p}
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </Grid>
+      <Grid container item xs={12} justify="flex-start">
+        <FormControlLabel
+          control={<Checkbox icon={<VisibilityIcon />} checkedIcon={<VisibilityOffIcon />} name="hidePractices" value={formState.hidePractices} onChange={handleBool} />}
+          label="Hide farming practices on marketplace"
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <FormControl className={classes.formControl} variant="outlined">
+          <InputLabel id="mutiple-name-label" required>Ownership and Operation</InputLabel>
```

Comment:
Ownership and operation shouldn't be required

Reply:
Hmm the figma and the design doc say its required though